### ISSUE TITLE
Adding more MDs for different languages

### ISSUE THREAD

**Issue by jkurz** [TurnerBroadcasting/MSS-Code-Guidelines#36]
This is a great project and i'm really happy we are starting up something like this. 

A couple of suggestions that I think would help keep this relevant. 

1. Split up READMEs into a section for JS, Python, Java, Ruby, and whatever else people want to use here. 
2. Split up JS into two sections. Client, Server
3. Make main README just links to these sections. 

I think this will allow for this to be extended further and utilized for more than just JS code.

**Comment by jamsyoung** [comment]
Yep, totally agree.  One thing to keep in mind.  Guidelines are worthless if they can't be enforced with a unit test. For other languages please include the config files for any tools like we have done for JavaScript and JSDoc with the dot files in this repo.  As more languages get added, we should probably get things out of the root of the repo as well and divide them into folders for languages.
